component-versions.md

```diff
@@ -2,6 +2,86 @@ This documents the versions of the different components for each Anbox Cloud rel
 
 Not all components are updated with each release. When components are not updated, this is called out in the [release notes](https://discourse.ubuntu.com/t/release-notes/17842) and components are marked with `n/a` below.
 
+[Details=1.18.0]
+
+### Charms
+
+#### Ubuntu 22.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.18/stable` | 116 |
+| `ams-node-controller` | `1.18/stable` | 125 |
+| `coturn` | `1.18/stable` | 119 |
+| `ams` | `1.18/stable` | 129 |
+| `aar` | `1.18/stable` | 122 |
+| `anbox-stream-gateway` | `1.18/stable` | 134 |
+| `ams-load-balancer` | `1.18/stable` | 124 |
+| `ams-lxd` | `1.18/stable` | 121 |
+| `anbox-stream-agent` | `1.18/stable` | 129 |
+| `nats ` | ` latest/stable` | 11 |
+
+#### Ubuntu 20.04
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud-dashboard` | `1.18/stable` | 115 |
+| `ams-node-controller` | `1.18/stable` | 124 |
+| `coturn` | `1.18/stable` | 118 |
+| `ams` | `1.18/stable` | 128 |
+| `aar` | `1.18/stable` | 121 |
+| `anbox-stream-gateway` | `1.18/stable` | 133 |
+| `ams-load-balancer` | `1.18/stable` | 123 |
+| `ams-lxd` | `1.18/stable` | 120 |
+| `anbox-stream-agent` | `1.18/stable` | 128 |
+| `nats ` | ` latest/stable` | 11 |
+
+### Validated revisions of 3rd party charms
+
+n/a
+
+### Bundles
+
+| Name | Channel | Revision |
+|----------|--------------|--------------|
+| `anbox-cloud` | `1.18/stable` | 78 |
+| `anbox-cloud-core` | `1.18/stable` | 83 |
```

Comment:
The revisions will change when we run the `promote-bundles` job today. So these are not final.